bitnami/contour/README.md

```diff
@@ -401,6 +401,16 @@ Find more information about how to deal with common errors related to Bitnami’
 
 Please carefully read through the guide "Upgrading Contour" at https://projectcontour.io/resources/upgrading/.
 
+### To 5.0.0
+
+In this version it was synchronized CRD with the official Contour repository https://github.com/projectcontour/contour/blob/main/examples/render/contour.yaml
```

Comment:
Could you change this Contour repository https://github.com/projectcontour/contour/blob/main/examples/render/contour.yaml to ⬇️?
```markdown
[Contour repository](https://github.com/projectcontour/contour/blob/main/examples/render/contour.yaml)
```
In that way, you can see that as a link: 
 [Contour repository](https://github.com/projectcontour/contour/blob/main/examples/render/contour.yaml)

Reply:
done